server/services/database/documents.js

```diff
@@ -110,6 +110,21 @@ const getDocumentIdByTranslationId = translationId => {
     .then(result => result.rows[0].document_id);
 };
 
+const deleteDocument = documentId => {
+  const query = "DELETE FROM documents WHERE id =$1";
+  return pool
+    .query(query, [documentId])
+    .then(result => result.rows)
+    .catch(error => console.error(error));
```

Comment:
Remove console.error here and catch here. You can catch the error in the route code to return the right status code and error message.

Reply:
ok